plugin-server/src/main/ingestion-queues/session-recording/utils.ts

```diff
@@ -128,9 +136,34 @@ export async function readTokenFromHeaders(
     return { token, teamIdWithConfig }
 }
 
+function readLibVersionFromHeaders(headers: MessageHeader[] | undefined): string | undefined {
+    const libVersionHeader = headers?.find((header: MessageHeader) => {
```

Comment:
Nit: You don't need to redeclare the type here (in fact you shouldn't as it can hide bugs)